spanner/spanner_arrays/README.md

```diff
@@ -0,0 +1,6 @@
+This is a demonstration of the use of Cloud Spanner arrays using the golang
+client library.  The key takeaway here is that even if your fields are not
+nullable, you still _must_ use the `spanner.NullString` type to receive them.
+Populating them can be done with either `spanner.NullString` _or_ plain
+`string`.
```

Comment:
Add a comment saying why? The string field may not be set in the row, so the value can be nil.

Reply:
Unfortunately, I don't _know_ why.  That's what makes this odd.  The field is set, for instance, with the `NOT NULL` restriction, but `spanner.NullString` is still required if that string field is part of an array.  Basically, if the _array_ is nullable, so must be the underlying scalar type (it would seem).